diff --git a/node_modules/@eslint/eslintrc/lib/config-array/override-tester.js b/node_modules/@eslint/eslintrc/lib/config-array/override-tester.js
index 3a445b1..afc8513 100644
--- a/node_modules/@eslint/eslintrc/lib/config-array/override-tester.js
+++ b/node_modules/@eslint/eslintrc/lib/config-array/override-tester.js
@@ -20,9 +20,7 @@
 import assert from "node:assert";
 import path from "node:path";
 import util from "node:util";
-import minimatch from "minimatch";
-
-const { Minimatch } = minimatch;
+import { minimatch, Minimatch } from "minimatch";
 
 const minimatchOpts = { dot: true, matchBase: true };
 
